app/src/main/res/drawable/ic_help_outlined.xml

```diff
@@ -0,0 +1,11 @@
+<vector
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="21dp"
+    android:height="20dp"
```

Comment:
invalid icon import, you should have 24x24

Reply:
this was just copy from onboarding branch

Reply:
looks like this wont work
![image](https://user-images.githubusercontent.com/99185280/195315846-175aa1a6-2a44-4c21-9cad-0bdf1ff48cb8.png)